kafka-ui-react-app/src/components/App.tsx

```diff
@@ -32,13 +32,12 @@ const App: React.FC = () => {
   const location = useLocation();
 
   React.useEffect(() => {
-    closeSidebar();
-  }, [closeSidebar, location]);
+    setIsSidebarVisible(false);
```

Comment:
Why not use closeSidebar function? If you want to prevent running useEffect on each render, instead of avoiding using closeSidebar you can wrap closeSidebar function In useCallback. Please, check it.